addons/live-featured-project/script.js

```diff
@@ -1,14 +1,20 @@
 export default async function ({ addon, msg }) {
   const showMenu = addon.settings.get("showMenu");
-  const forceAlternative = addon.settings.get("forceAlternative");
-  const alternativePlayer = addon.settings.get("alternativePlayer");
+  const player = addon.settings.get("alternativePlayer");
   const autoPlay = addon.settings.get("autoPlay");
   const enableTWAddons = addon.settings.get("enableTWAddons");
-  const enabledAddons = await addon.self.getEnabledAddons("editor");
+  const shareUsername = addon.settings.get("shareUsername");
 
   const stageElement = document.querySelector(".stage");
   const projectId = window.Scratch.INIT_DATA.PROFILE.featuredProject.id;
 
+  // Check if project is unshared before doing anything
+  const featuredProject = await fetch(`https://api.scratch.mit.edu/projects/${projectId}`);
```

Comment:
Could you use the `HEAD` method? It should you save some bytes (with negligble speed difference), since you only check for the status.

```suggestion
  const featuredProject = await fetch(`https://api.scratch.mit.edu/projects/${projectId}`, { 'method': 'HEAD' });
```

Reply:
PS. Make sure to test it! I haven't run the code in whole!

Reply:
Just did on one shared and one unshared project and it seems to work.